PokeAppAndroid/Adapters/PokemonAdapter.cs

```diff
@@ -25,7 +35,26 @@ public override int ItemCount
         public override void OnBindViewHolder(RecyclerView.ViewHolder holder, int position)
         {
             PokemonViewHolder viewHolder = holder as PokemonViewHolder;
-            viewHolder.TvPokemonName.Text = pokemomList[position].name;
+            var value = pokemomList[position];
+            string pokemonID = value.ID;
+
+            viewHolder.Container.SetDrawableBackgroundForType(value.Types[0]);
```

Comment:
As a suggestion maybe you can use Types.First() or Types.FirstOrDefault() instead of Types[0]